ffi/examples/read-table/schema.h

```diff
@@ -166,7 +167,8 @@ void visit_map(
 {
   SchemaBuilder* builder = data;
   char* name_ptr = malloc(sizeof(char) * (name.len + 22));
-  snprintf(name_ptr, name.len + 1, "%s", name.ptr);
+  int name_chars = name.len > INT_MAX ? INT_MAX : (int)name.len; // handle _REALLY_ long names by truncation
+  snprintf(name_ptr, name.len + 1, "%.*s", name_chars, name.ptr);
```

Comment:
Should this instead be `name.len + 22` (since the size of buffer is that but we will write less?)